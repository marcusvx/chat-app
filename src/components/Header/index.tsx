import { Navbar } from "react-bulma-components";
import icon from "./chat-icon.png";

const Header = () => {
  return (
    <Navbar className="has-shadow">
      <Navbar.Brand>
        <Navbar.Item href="/">
          <img alt="A chat room app" height="32" src={icon} width="32" className="m-2" />
          <h1 className="is-size-4 is-uppercase has-text-weight-bold">
            Chat Rooms
          </h1>
        </Navbar.Item>
        <Navbar.Burger />
      </Navbar.Brand>
    </Navbar>
  );
};

export default Header;
