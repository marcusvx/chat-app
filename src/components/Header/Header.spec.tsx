import { render } from "@testing-library/react";
import Header from "./Header";

describe("Header Component", () => {
  it("should match snapshot", () => {
    expect(render(<Header />)).toMatchSnapshot();
  });
});
