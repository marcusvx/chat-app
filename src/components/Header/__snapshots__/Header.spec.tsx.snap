// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component should match snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <nav
        class="navbar has-shadow"
        role="navigation"
      >
        <div
          class="navbar-brand"
        >
          <a
            class="navbar-item"
            href="/"
          >
            <img
              alt="A chat room app"
              class="m-2"
              height="32"
              src="chat-icon.png"
              width="32"
            />
            <h1
              class="is-size-4 is-uppercase has-text-weight-bold"
            >
              Chat Rooms
            </h1>
          </a>
          <div
            class="navbar-burger"
            role="button"
            style="outline: none;"
            tabindex="0"
          >
            <span />
            <span />
            <span />
          </div>
        </div>
      </nav>
    </div>
  </body>,
  "container": <div>
    <nav
      class="navbar has-shadow"
      role="navigation"
    >
      <div
        class="navbar-brand"
      >
        <a
          class="navbar-item"
          href="/"
        >
          <img
            alt="A chat room app"
            class="m-2"
            height="32"
            src="chat-icon.png"
            width="32"
          />
          <h1
            class="is-size-4 is-uppercase has-text-weight-bold"
          >
            Chat Rooms
          </h1>
        </a>
        <div
          class="navbar-burger"
          role="button"
          style="outline: none;"
          tabindex="0"
        >
          <span />
          <span />
          <span />
        </div>
      </div>
    </nav>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
